platform/libs.osgi/external/osgi-7.0-license.txt

```diff
@@ -1,6 +1,6 @@
 Name: OSGi
 Version: 7.0
-Files: osgi.core-7.0.0.jar org.osgi.compendium-5.0.0.jar
+Files: osgi.core-7.0.0.jar osgi.cmpn-7.0.0.jar
```

Comment:
Testing this change in [OracleLabs IGV](https://www.graalvm.org/docs/graalvm-as-a-platform/implement-language/#igv) revealed that the `org.osgi.service.resolver.ResolveContext` is present twice:
* in the OSGi core JAR
* in the OSGi companion JAR

when the companion version took preference, the Felix boot failed due to linkage error as `onCancel(Runnable)` method was missing. Updating to `7.0.0` (e.g. the same version as the OSGi core) should solve this inconsistency.